plugins/janus_videoroom.c

```diff
@@ -632,6 +632,20 @@ room-<unique room ID>: {
 	"videoroom" : "event",
 	"configured" : "ok"
 }
+\endverbatim
+ *
+ * When configuring the room to request the ssrc-audio-level RTP
+ * extension, we saw that events could be sent to the publishers if
+ * \c audiolevel_event is set to true. These events will have the
+ * following format:
```

Comment:
This text ("we saw...") doesn't work for the documentation: here the feature must be explained, not "discovered". It should look something like this instead:

```
When configuring the room to request the ssrc-audio-level RTP
extension, ad-hoc events might be sent to all publishers if
\c audiolevel_event is set to true. These events will have the
following format:
```

Reply:
done.